cmd/kyverno/main.go

```diff
@@ -291,8 +291,9 @@ func main() {
 		kyvernoV1.ClusterPolicies(),
 		kyvernoV1.Policies(),
 		kyvernoInformer.Kyverno().V1beta1().UpdateRequests(),
-		eventGenerator,
 		kubeInformer.Core().V1().Namespaces(),
+		kubeInformer.Core().V1().Pods(),
```

Comment:
We cannot use the pod informer as Kyverno memory grows if pods number increases.

Reply:
cc @eddycharly @prateekpandey14